testsuite/update-dump.cmake

```diff
@@ -12,18 +12,17 @@ foreach(TEST_FOLDER ${TEST_FOLDERS})
       set(OUT_FILE ${TEST_FOLDER}/ast-parse/${TEST_NAME}/ast.txt)
       message(STATUS "Regenerating ${OUT_FILE}")
       execute_process(
-        COMMAND ${PARSER} -a -g ${GRAMMAR} ${PARSE_TEST}
+        COMMAND ${VERONA_AST} -a -g ${GRAMMAR} ${PARSE_TEST}
         OUTPUT_FILE ${OUT_FILE})
     endforeach()
   elseif(${TEST_FOLDER} MATCHES ".*/mlir")
     file(GLOB MLIR_TESTS "${TEST_FOLDER}/mlir-parse/*.verona" "${TEST_FOLDER}/mlir-parse/*.mlir")
     foreach(MLIR_TEST ${MLIR_TESTS})
       get_filename_component(TEST_NAME ${MLIR_TEST} NAME_WE)
-      set(OUT_FILE ${TEST_FOLDER}/mlir-parse/${TEST_NAME}/out.mlir)
+      set(OUT_FILE ${TEST_FOLDER}/mlir-parse/${TEST_NAME}/mlir.txt)
```

Comment:
Using `out.mlir` will mean VS code will syntax highlight more nicely.  Would it be better to switch the other way?

Reply:
An issue with out.mlir is that the copyright check script will look at these files and fail. Also the check_dump script is configured to only look at .txt files.

These are both fixable issues, but naming it mlir.txt sounds like the simplest option.

Reply:
They sounds good reasons.  So please ignore.

Reply:
Ah, I changed this on the test runner and forgot to update the dump. This change is correct, even though I agree with @mjp41 's points. :)